lib/flight_job/models/job.rb

```diff
@@ -390,8 +394,8 @@ def submit
         # Parse stdout on successful commands
         process_output('submit', status, out) do |data|
           self.scheduler_id = data['id']
-          self.stdout_path = data['stdout']
-          self.stderr_path = data['stderr']
+          self.stdout_path = data['stdout'].blank? ? "/dev/null" : data['stdout']
+          self.stderr_path = data['stderr'].blank? ? "/dev/null" : data['stderr']
```

Comment:
I presume something breaks if we use `nil` instead of `/dev/nuill`.  I think I'd prefer the UI to state that the stdout wasn't available rather than this

![image](https://user-images.githubusercontent.com/287050/130051574-8ac2c184-4a46-4610-93e8-0755ee228ab7.png)

I can handle that client-side if its too awkward to do here.

Reply:
My mistake, I am starting to loose track of `required` keys within the `--json` output. On review, the `stdout_path`/`stderr_path` are actually "recommended":

30a64c8f807c41840a0da13245c9387029e7a1a8

Reply:
Thanks for that @WilliamMcCumstie .  We now have the following which is closer to what has happened.

![image](https://user-images.githubusercontent.com/287050/130097690-f2261a70-3cd5-47d0-ac15-13942063e649.png)